spec/default_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 describe 'haproxy::default' do
-  let(:chef_run) { ChefSpec::Runner.new.converge(described_recipe) }
+  let(:chef_run) { ChefSpec::SoloRunner.new(platform: 'ubuntu', version: '16.04').converge(described_recipe) }
```

Comment:
it was agnostic, and I got this warning: `haproxy::default
WARNING: you must specify a platform and platform_version to your ChefSpec Runner and/or Fauxhai constructor, in the future omitting these will become a hard error`. I was trying to clean it up. Should I revert?